src/Resources/config/routing.yml

```diff
@@ -11,3 +11,9 @@ whatwedo_crud_dashboard:
     controller: Symfony\Bundle\FrameworkBundle\Controller\TemplateController
     defaults:
         template: '@whatwedoCrud/dashboard.html.twig'
+
+whatwedo_crud_search:
+    path: /whatwedo_crud_search
+    controller: whatwedo\CrudBundle\Controller\SearchController::search
+    defaults:
+        permanent: true
```

Comment:
`permanent: true` is used for redirects: https://symfony.com/doc/current/routing.html#redirecting-to-urls-and-routes-directly-from-a-route

remove